sites/docs/src/content/docs/tutorials/external_usage/nf-core_configs_outside_nf-core.md

```diff
@@ -7,84 +7,128 @@ When you've been working with nf-core pipelines for a while, you often will get
-
-```groovy
-// Load nf-core custom profiles from different Institutions
-try {
-    includeConfig "https://raw.githubusercontent.com/nf-core/configs/master/nfcore_custom.config"
-} catch (Exception e) {
-    System.err.println("WARNING: Could not load nf-core/config profiles: https://raw.githubusercontent.com/nf-core/configs/master/nfcore_custom.config")
-}
-```
-
-Note nf-core pipelines use a [parameter for the URL](https://github.com/nf-core/tools/blob/0912990a63ef29e44e07cc2ba6ab81113684e0ae/nf_core/pipeline-template/nextflow.config#L67-L72) to allow debugging.
-
-4. Finally Add the the [`check_max`](https://github.com/nf-core/tools/blob/0912990a63ef29e44e07cc2ba6ab81113684e0ae/nf_core/pipeline-template/nextflow.config#L233-L264) function to `nextflow.config`
-
-The following should be placed outside all scopes, typically at the bottom of the file:
-
-```groovy
-// Function to ensure that resource requirements don't go beyond
-// a maximum limit
-def check_max(obj, type) {
-    if (type == 'memory') {
-        try {
-            if (obj.compareTo(params.max_memory as nextflow.util.MemoryUnit) == 1)
-                return params.max_memory as nextflow.util.MemoryUnit
-            else
-                return obj
-        } catch (all) {
-            println "   ### ERROR ###   Max memory '${params.max_memory}' is not valid! Using default value: $obj"
-            return obj
-        }
-    } else if (type == 'time') {
-        try {
-            if (obj.compareTo(params.max_time as nextflow.util.Duration) == 1)
-                return params.max_time as nextflow.util.Duration
-            else
-                return obj
-        } catch (all) {
-            println "   ### ERROR ###   Max time '${params.max_time}' is not valid! Using default value: $obj"
-            return obj
-        }
-    } else if (type == 'cpus') {
-        try {
-            return Math.min( obj, params.max_cpus as int )
-        } catch (all) {
-            println "   ### ERROR ###   Max cpus '${params.max_cpus}' is not valid! Using default value: $obj"
-            return obj
-        }
-    }
-}
-```
+Here we will describe the steps you will need to perform in your custom script or pipeline to use nf-core institutional configs.
+
+1. In a `conf/base.config` file set default basic resources for all processes. Can be named something else.
+
+   At the bare minimum, the file should contain:
+
+   ```groovy
+   process {
+       cpus   = 1
+       memory = 7.GB
+       time   = 4.h
+   }
+   ```
+
+   (or other sensible default values)
+
+   :::note{collapse title="Note on older nf-core template/Nextflow versions"}
+
+   If you wish to use pipelines generated with the pre-`v3.0.0` nf-core template, and/or when running with Nextflow versions earlier than 24.04.0 you may need to use an older syntax for setting resources. The following prevents the resources from exceeding a maximum limit.
```

Comment:
```suggestion
   When running pipelines generated with the nf-core template pre-version v3.0.0 or with Nextflow pre-version 24.04.0 you may need to use an older syntax for setting resources. The following closures prevent resources from exceeding a maximum limit.
```

Reply:
I found this quite hard to follow, feel free to ignore if you disagree.

Reply:
I see what you mean but I think putting nf-core/nextflow before makes `pre-XXX` read as an actual version, rather than saying beforee that.

Maybe I just replace pre- with 'before version'...